pkg/deploy/predeploy.go

```diff
@@ -229,6 +229,9 @@ func (d *deployer) ensureEncryptionSecret(ctx context.Context, serviceKeyVaultUR
 	d.log.Infof("setting %s", env.EncryptionSecretName)
 	_, err = d.keyvault.SetSecret(ctx, serviceKeyVaultURI, env.EncryptionSecretName, keyvault.SecretSetParameters{
 		Value: to.StringPtr(string(key)),
+		SecretAttributes: &keyvault.SecretAttributes{
+			Enabled: to.BoolPtr(true),
```

Comment:
I'm amazed this doesn't happen by default

Reply:
yeah...

Reply:
it does; this PR is wrong.